tests/partners/factories.py

```diff
@@ -21,3 +23,11 @@ def admins(self, create, extracted, **kwargs):
         if extracted:
             for user in extracted:
                 self.admins.add(user)
+
+    @classmethod
+    def _create(cls, model_class, *args, **kwargs):
```

Comment:
why didn't you use something like the postgeneration `make_mbox` example in the docs?
http://factoryboy.readthedocs.io/en/latest/reference.html#factory.PostGeneration

```
    @factory.post_generation
    def auto_set_partner(self, create, extracted, **kwargs):
        if extracted is None or extracted:
            set_partner(self)
```

Reply:
As far as understood, the post_generation annotation works on a field level and has to be called manually.